scripts/extensions/sams/src/components/assets/assetEditor.tsx

```diff
@@ -24,11 +27,34 @@ interface IProps {
     sets: Array<ISetItem>;
     fields?: Array<keyof IAssetItem>;
 }
+interface ITagUi {
```

Comment:
You could use `IAssetItem['tags']` here instead of re-declaring the same interface